hack/make/.integration-daemon-start

```diff
@@ -6,6 +6,10 @@ base="$ABS_DEST/.."
 export PATH="$base/binary-daemon:$base/dynbinary-daemon:$PATH"
 
 export TEST_CLIENT_BINARY=docker
+
+# Do not bump this version! Integration tests should no longer rely on the docker cli, they should be
+# API tests instead. For the existing tests the scripts will use a frozen version of the docker cli
+# with a DOCKER_API_VERSION frozen to 1.30, which should ensure that the CI remains green at all times.
```

Comment:
Updated  👍  - also added this note to prevent people from changing it in future :smile: